docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
+                                timezone=timezone)  
+    >>> output = detector.detect(message=message, entity_name=entity_name,
+                                 structured_value=structured_value,
+                                 fallback_value=fallback_value)
+    >>> print(output)
+    ```
+
+  - *CURL command*
+
+    ```bash
+    URL='localhost'
+    PORT=8081
+    
+    curl -i 'http://'$URL':'$PORT'/v2/time/?message=राजू%20का%20बस%20१३:५०%20को%20बस%20स्टॉप%20से%20निकला%20और%20१५%20मिनट%20में%20यहाँ%20पहुंच%20जाएगा%20और%20गोवा%20को%20शाम%20में%20बारह%20बजकर%20३०%20मिनट%20पैर%20पहुंचेगा&entity_name=time&structured_value=&fallback_value=&bot_message=&timezone=UTC&source_language=en'
+    
+    ```
+
+    > **Output**:
+
+    ```json
+    {"data": [
+        {
+           "detection": "message",
+           "original_text": "१३:५०",
+           "entity_value": { "mm": 1, "hh": 50,"nn": "hr"},
+            "language": "hi"
+        },
+        {
             "detection": "message",
-            "original_text": "mummbai",
-            "entity_value": {
-              "to": false,
-              "via": false,
-              "from": true,
-              "value": "Mumbai",
-              "normal": false
-            }
-          }
-        ]
-      }
-      ```
-
-### pnr
-
--   This functionality calls the PNRDetector class to detect pnr.
-
-- Example:
-
-- - Example 1: 
-
-    - ```python
-      message = 'check my pnr status for 2141215305.'
-      entity_name = 'train_pnr'
-      structured_value = None
-      fallback_value = None
-      bot_message = None
-      ```
-
-    - *Python:*
-
-      ```python
-      from ner_v1.chatbot.entity_detection import get_pnr
-      output = get_pnr(message=message, entity_name=entity_name,structured_value=structured_value,fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
-
-    - *CURL command:*
-
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
-
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/pnr/?message=check%20my%20pnr%20status%20for%202141215305.&entity_name=pnr&structured_value=&fallback_value=&bot_message='
-      ```
-
-    - *CURL Output:*
-
-      ```json
-      {
-        "data": [
-          {
+            "original_text": "१५ मिनट में",
+            "entity_value": {"mm": "15", "hh": 0, "nn": "df"},
+          "language": "hi"
+        },
+        {
+           "detection": "message",
+           "original_text": "शाम में बारह बजकर ३० मिनट",
+           "entity_value": { "mm": 30, "hh": 12, "nn": "pm"},
+            "language": "hi"
+        }]
+    }
+    
+    ```
+
+    
+
+### 2. Date
+
+> entity_type: date
+
+The Date detector module has the capability to detect various form of dates from text in multiple languages. It can detect date from following patterns:
+
+1. *Day month year format*  - 12 feb 2018, 2nd Jan 2019,  12/11/2019, 12-jan-2019
+2. *Day month* - 12 feb, 12/12
```

Comment:
Formatting inconsistency
points 1 and 2 are in Italics, rest are not. Also, I might be opinionated but bold looks better than italics in this case